spec/features/super-admin/view_organisation_page_spec.rb

```diff
@@ -75,6 +60,27 @@
       end
     end
 
+    context 'with five recent sessions with different usernames' do
+      before do
+        ('A'..'E').each do |char|
+          Session.create(
+            siteIP: create(:ip, location: location).address,
+            success: true,
+            username: char * 6,
+            start: Time.now.to_s
+          )
+        end
+      end
+
+      it 'shows five unique connections' do
+        visit admin_organisation_path(organisation)
+
+        within('#unique-connections') do
+          expect(page).to have_content('5')
```

Comment:
Think this could easily be a false positive if there was a 5 anywhere on the page.
Would consider targeting a specific piece of markup with "within", or refining the match string.